Expanding Space/Assets/Scripts/Dialog/DialogManager.cs

```diff
@@ -0,0 +1,140 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class DialogManager : MonoBehaviour
+{
+    public TextMeshProUGUI dialogText;
+    public GameObject continueImage;
+
+    public GameObject DialogHolder;
```

Comment:
Maybe change the capital to a small letter

Reply:
Dat klopt! Ik ga het meteen aanpassen.